pkg/agent/sysadvisor/plugin/qosaware/server/memory_server.go

```diff
@@ -64,26 +65,26 @@ func (ms *memoryServer) ListAndWatch(_ *advisorsvc.Empty, server advisorsvc.Advi
 	for {
 		select {
 		case <-ms.stopCh:
-			general.Infof("lw stopped because %v stopped", ms.name)
+			klog.Infof("[qosaware-server-memory] lw stopped because %v stopped", ms.name)
```

Comment:
why we should switch general log back to klog ...

Reply:
Prefix is needed for more efficient debug; Package name logged by general decreases readability.

Reply:
> Prefix is needed for more efficient debug. Package name logged by general decreases readability.

all right... still kind of ugly to have so many [xxx] spread everywhere ... i will not insist, but i have to point this coding style...

Reply:
trade off between quicker bug location by log & slight better code style : )